### PR TITLE
impr(quotes): created and added quote to english.json (ElijahRosal)

### PR DESCRIPTION
**Pull request by ElijahRosal**
### Description

Added a quote to english.json
### Checks

- [ ] Adding quotes?
  - [ ] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [ ] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [ ] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [ ] Make sure to include your GitHub username inside parentheses at the end of the PR title

Closes #

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/9024264714

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38615,6 +38615,13 @@
       "source": "Leto Atreides II, God Emperor of Dune",
       "length": 107,
       "id": 7058
-    }
+    },
+    {
+      "text": "Oh. Round and round we go. Holding on to pain drivеn by our egos. Feelings untold. Spinning our own versions of the past that you could nеver know cause I know.",
+      "source": "Mili - Through Patches of Violet",
+      "length": 160,
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38622]: `Spinning our own versions of the past that you could nеver know cause I know.` i feel like there is a comma missing here somewhere

**Review by Miodec — CHANGES_REQUESTED** [review]

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/9067363908

**Review by Miodec — APPROVED** [review]

**Comment by Miodec** [comment]
Thanks

**Pull request merged**